extension/src/tree/index.ts

```diff
@@ -47,7 +47,8 @@ export const getCliErrorLabel = (msg: string): string =>
 export const getCliErrorTreeItem = (
   label: string,
   msg: string,
-  decoratableTreeItemScheme: DecoratableTreeItemScheme
+  decoratableTreeItemScheme: DecoratableTreeItemScheme,
+  isLabelPath = true
```

Comment:
`isLabelPath` is hard to track. 

IIUC -
You are setting it to true for both `repository` and `experiments` but the default will also set it to true for `plots`.

then `!isLabelPath` will then never get hit.

Reply:
can you add some tests around this please

Reply:
> You are setting it to true for both repository and experiments but the default will also set it to true for plots.

Whoops, forgot to push the latest commit that fixed that.

> isLabelPath is hard to track.

Makes sense, I'll rethink the solution. 

> can you add some tests around this please

I wasn't sure how to test for this, but after reviewing our tree tests again, maybe I can test with `getTreeItem`. I'll look into that!